Zend/zend.c

```diff
@@ -119,6 +119,17 @@ static ZEND_INI_MH(OnUpdateErrorReporting) /* {{{ */
 }
 /* }}} */
 
+static ZEND_INI_MH(OnUpdateErrorBacktraceRecording)
+{
+	if (!new_value) {
+		EG(error_backtrace_recording) = E_FATAL_ERRORS;
```

Comment:
I think no value should be disabled and you should set INI default to fatal if you really want it by default on fatal. I would personally prefer to disalbe by default completely